px-context-browser.html

```diff
@@ -1025,6 +1129,71 @@ <h1 class="epsilon caps weight--normal u-m0" on-click="toggleColumnBrowser">
         if (root && root.render) {
           root.render();
         }
+      },
+
+      /**
+       * Called when the checkboxes are clicked
+       * @private
+       *
+       */
+      _itemsSelected: function(evt) {
+        evt.stopPropagation();
+        if (evt.target.checked && this.limit !== 0 && this.limit === this.selectedItems.length) {
+          evt.preventDefault();
+          return false;
+        }
+
+        if (evt.target.checked) {
+          this.selectedItems.push(evt.model.item);
+        } else {
+          _.pull(this.selectedItems, evt.model.item);
```

Comment:
Two issues here:

1. This is using the lodash `_.pull` method. This component doesn't include lodash as a dependency. If the app developer doesn't include lodash in the app's JavaScript alongside the px-context-browser import, this method throws a TypeError and fails out. We usually try to avoid shipping big helper libraries that will muck up the `window` globals. It might make sense to just use native JavaScript to do this.
2. If you do end up using `_.pull` or implementing that method yourself, mutating the `selectedItems` array like this will not trigger property binding notifications/events on the `selectedItems` property. You'll need to manually calculate splices and call them with Polymer's [`notifySplices`](https://www.polymer-project.org/1.0/docs/api/classes/Polymer.Base#method-notifySplices) API, or use the Polymer `this.splice` API to ensure property change bindings are triggered.